fonts.css

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: "GothamPro";
+  src: 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5896505)